packages/edge-config/package.json

```diff
@@ -49,6 +49,7 @@
     "@changesets/cli": "2.26.2",
     "@edge-runtime/jest-environment": "2.3.6",
     "@edge-runtime/types": "2.2.6",
+    "@opentelemetry/api": "1.7.0",
```

Comment:
Did we have a goal to make this a peer dependency?

Reply:
I guess that would make more sense? Since we only need the type really 🤔

Reply:
it shouldn't be a peer dep since that would require people to install it and lead to ugly warnings. I think it could potentially be a devDependency if the type gets bundled correctly?

Reply:
Probably, but seems like it doesn't get bundled, at least`index.d.ts` tries to import from `@opentelemetry/api`.

Reply:
It should be a peer dependency indeed.

Reply:
I've been kind of going back and force on this. But I believe this should be a peer dependency in the end. Upon further investigation, a peer dependency doesn't requires a parent to install. The parent only need to install if they want to use `@opentelemetry/api` themselves, which is exactly what we want here.

Reply:
+1 for marking as a peer dependency, currently v1.0.0 breaks on edge runtime environments (see #606)